libraries/Microsoft.Bot.Builder.AI.LanguageGeneration/Evaluator.cs

```diff
@@ -128,13 +131,33 @@ public override string VisitNormalTemplateString([NotNull] LGFileParser.NormalTe
                         builder.Append(EvalMultiLineText(node.GetText()));
                         break;
                     default:
-                        builder.Append(node.GetText());
+                        builder.Append(EvalEscapeCharacter(node.GetText()));
```

Comment:
Why do you need to change the default behavior of evalution?

Reply:
I have deleted it at the newest version